<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpWindow</class>
 <widget class="QMainWindow" name="HelpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>354</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>403</width>
    <height>354</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/appLogo/icon.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 161, 67);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="headingIns">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 161, 67);</string>
    </property>
    <property name="text">
     <string>Instructions</string>
    </property>
   </widget>
   <widget class="QLabel" name="help1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 161, 67);</string>
    </property>
    <property name="text">
     <string>1. Generate QRCode: Click on the Generate QRCode Button and then a new window will open there enter the data and click on Save Button. Automatically QRCode of the entered data will be generated and save at that location.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="help2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 161, 67);</string>
    </property>
    <property name="text">
     <string>2. Read QRCode From Image: Click on the Read QRCode Button and then a new window will open browse to the required image file and open it in the application. Then click on the Read QRCode Button to get the data.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="help2_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 161, 67);</string>
    </property>
    <property name="text">
     <string>2. Read QRCode Using Camera: First Select Source if you are using your first webcam then select 0 and you are using any other camera then select 1. Then a video window will pop up and when the data from the QR Code appear on the window press 'G' key on the keyboard to get the data.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
